src/packages/README.md

```diff
@@ -30,33 +103,67 @@ the [CheriBSD ports issue tracker](https://github.com/CTSRD-CHERI/cheribsd-ports
+have not been audited and only a limited set have been tested. Bugs related to
+CHERI support can be reported in the [CheriBSD ports issue
+tracker](https://github.com/CTSRD-CHERI/cheribsd-ports/issues).
+While the CheriABI packages are more interesting as they use CHERI memory-safety
+features, you must remember that they might break in run-time and hence might
+not be suitable for critical operations.
+In such case, a corresponding hybrid ABI package might be considered instead of
+a CheriABI package.
 
-## Managing packages in CheriBSD
+## Duplicate packages
 
-*Note:* As of this writing we only provide packages for Morello systems.
-We aim to add RISC-V package sets in the near future.
+Hybrid ABI and CheriABI package repositories include many duplicate packages,
+e.g. `git`.
+If you decided to use a CheriABI package and found out that it crashes, you
+might consider using a hybrid ABI package instead.
 
-Hybrid and CheriABI packages can be installed via two independent instances of
-the package system in CheriBSD, one for each ABI.
-The intention is that, over time, the CheriABI packages will become more
-mature, and hence the preferred collection for day-to-day use.
+You must remember that CheriABI packages have a higher priority than the hybrid
+ABI packages in `PATH` environment variables in CheriBSD.
+In case you installed a CheriABI package and a hybrid ABI package with a
+conflicting program name, you must execute the hybrid ABI program using
+an absolute path.
 
-The default `pkg` command will print a note describing this arrangement, but
-cannot be used for package management as described in FreeBSD documentation.
-To manage and install hybrid packages, use the `pkg64` command.
-To manage and install CheriABI packages, use the `pkg64c` command.
-We expect that `pkg64c` will be renamed to `pkg` in a future CheriBSD release.
+## Cross-ABI package dependencies
 
-You can learn more about FreeBSD's package manager by reading the [pkg(8)
-manual page](https://www.freebsd.org/cgi/man.cgi?pkg(7))
-and the [FreeBSD Handbook chapter on packages and
-ports](https://docs.freebsd.org/en/books/handbook/ports/).
+At the moment, it is not possible to provide packages with cross-ABI run-time
+package dependencies.
+Such feature would be useful if a CheriBSD port cannot be easily adapted to
+CheriABI and includes a program that is executed by another port that has
+a CheriABI package.
+There are currently no plans to support this case.
+
+## Missing packages
+
+Hybrid ABI and CheriABI package repositories might be missing third-party
+software due to the following reasons:
+* There is no FreeBSD port including that software or it is broken on AArch64;
+
+  Check [FreeBSD Wiki](https://wiki.freebsd.org/WantedPorts),
+  [FreeBSD Bugzilla](https://bugs.freebsd.org/bugzilla/) and
+  the [freebsd-ports mailing
+  list](https://lists.freebsd.org/subscription/freebsd-ports) to find out
+  if anyone is working on software you are interested in.
```

Comment:
It might be helpful to point people at FreshPorts which is where I go to see if something is ported. It's often the case that the pkg name just isn't obvious.

Reply:
Done.